src/core/audio/AudioSoundIo.cpp

```diff
@@ -205,18 +212,34 @@ AudioSoundIo::~AudioSoundIo()
 
 void AudioSoundIo::startProcessing()
 {
+	int err;
+	
 	m_outBufFrameIndex = 0;
 	m_outBufFramesTotal = 0;
 	m_outBufSize = mixer()->framesPerPeriod();
 
 	m_outBuf = new surroundSampleFrame[m_outBufSize];
 
+	if (! m_started)
```

Comment:
**No** ,` m_started` became true only once in object life. May be should be better name : `m_device_started `.

But now I think: What happens if ` m_started` will not became true? 
Should other code in start/stop methods be executed if `m_started == false` ?  
It seems that should - `m_stopped == true` will be in this case , and be printed error message. 

`m_started`  can be not introduced, but than should be found safe place for `soundio_outstream_start(m_outstream)`

**P.S.**

`m_stopped` - mean "_processing stopped_";
`m_started `- mean "_device started_".

**Should I rename** ` m_started` to (for example) `m_device_started ` ?

Reply:
Yeah, to me it feels like `m_deviceStarted`* would be clearer than just `m_started`. I'd like to defer the decision to someone more familiar with this part of the code though.

*LMMS conventions [specify](https://github.com/LMMS/lmms/wiki/Coding-Conventions#variable-names) that variables should be camelCase, rather than snake_case.

Reply:
I decided to change variable name to `m_outstreamStarted`  :
```
	if (! m_outstreamStarted)
	{
		if ((err = soundio_outstream_start(m_outstream)))
```
P.S.
It seems, that after active conversation all is stopped. May be it is normal and I should wait, 
but may be You are waiting for some my "job", but I do not understand this.

There for I write this message ...